codespell_lib/data/dictionary.txt

```diff
@@ -7461,11 +7461,17 @@ obay->obey
 obect->object
 obediance->obedience
 obediant->obedient
+obejct->object
 obeserve->observe
 obession->obsession
 obgect->object
+objectss->objects
 objekt->object
+objetcs->objects
+objext->object
```

Comment:
I wonder if this shouldn't be auto-corrected due to OBJEXT ( http://www.gnu.org/software/automake/manual/html_node/LIBOBJS.html ) in Automake, the same as some C stuff has a comment